### ISSUE TITLE
Mixins being loaded too early

### ISSUE THREAD

**Issue by Sunconure11** [KubeJS-Mods/Rhino#25]
https://pastebin.com/nNc3p700

**Comment by Sunconure11** [comment]
This happens on Quilt 1.19

**Comment by Sunconure11** [comment]
@MaxNeedsSnacks

**Comment by ProsperCraft** [comment]
Same on Forge 1.19
rhino-forge-1802.1.14-build.206
Forge 41.0.110

Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTargetAlreadyLoadedException: Critical problem: rhino-common.mixins.json:ChatFormattingMixin target net.minecraft.ChatFormatting was loaded too early.

Debug log-
https://gist.githubusercontent.com/ProsperCraft/57650c9e3f3087662f7f5a240832ae8f/raw/5c2f417ab810ff52de5aa83292593346476752e0/gistfile1.txt

Console showing error-
https://gist.githubusercontent.com/ProsperCraft/b8a10a0173dabaeee32f70c3b443429f/raw/f28e5f617373c333de35cf3d8ca7cb33702bef59/gistfile1.txt

**Comment by Sunconure11** [comment]
I did some talking around and it is likely that someone is trying to load this class in a mixin plugin.

**Comment by MaxNeedsSnacks** [comment]
Yes, I would've suspected either that or some other ASM / early riser thing... The question is, what mod would need to do that

**Comment by Sunconure11** [comment]
Is there some way of either adding a failsafe or a workaround in this mod to possibly expose the culprit?

**Comment by MaxNeedsSnacks** [comment]
Not really, since the mod won't even start loading in the first place. You'll have to find whichever mod loads that class unfortunately

**Comment by Sunconure11** [comment]
I did go through a bunch of other mods, and eventually led to the one causing an unrelated crash. I wonder if one of them could be at play here. I will send you the names of the mods and associated logs.

Creeper Overhaul
Biome Makeover
Terrablender
Arcanus
Spoorn Pink
Spoorn Packs
Left Hanging
Reaping
MC Dungeons + Story suite
Requiem
Enhanced Block Entities
Better End?

https://pastebin.com/eqe3rC8Z

https://pastebin.com/3s2edtRz

https://pastebin.com/NCKULuSb

https://pastebin.com/8m8811wf

https://pastebin.com/dHELGFW2

https://gist.github.com/Sunconure11/fe55897fbe7e77d05dce12e5d8c1e14e

https://gist.github.com/Sunconure11/7abf61efa57e81f29264af29506bc074

https://gist.github.com/Sunconure11/58d82e41bddef982195cca3cc12ecdd7

**Comment by Sunconure11** [comment]
The mod causing this is FPS Reducer.

https://www.curseforge.com/minecraft/mc-mods/fps-reducer

**Comment by MaxNeedsSnacks** [comment]
What kinda business do they have loading ChatFormatting early.... odd

**Comment by MaxNeedsSnacks** [comment]
Marking as wontfix because... well, we can't, this'll have to be fixed on their end since they should not be loading classes before Mixins have been applied
